docs/index.rst

```diff
@@ -11,7 +11,6 @@ Features
 * :ref:`sync-publish-workflow` with "RPM Content" including RPMs and Errata
 * :ref:`Versioned Repositories <versioned-repo-created>` so every operation is a restorable snapshot
 * :ref:`Download content on-demand <create-remote>` when requested by clients to reduce disk space.
-* Upload local RPM content :ref:`easily <one-shot-upload-workflow>`
```

Comment:
@pavelpicka what about leaving the upload feature where we can say "Upload local RPM content in chunks'

Reply:
I added link to generic upload from pulpcore which describes both (upload, chunked upload) methods.